widgets/button/button.php

```diff
@@ -231,14 +243,15 @@ function get_template_variables( $instance, $args ) {
 
 		$attributes = $instance['attributes'];
 
+		// 'icon_placement' => $instance['button_icon']['icon_placement'],
+
 		$classes = ! empty( $attributes['classes'] ) ? $attributes['classes'] : '';
+		$classes .= ' ow-icon-placement-'. $instance['button_icon']['icon_placement'];
 		if( !empty($instance['design']['hover']) ) {
 			$classes .= ' ow-button-hover';
 		}
 
-		if( ! empty( $classes ) ) {
-			$button_attributes['class'] = $classes;
-		}
+		$button_attributes['class'] = $classes;
```

Comment:
We're already passing this through esc_attr in the button template, but maybe we should also pass it through sanitize_html_class first just to make sure sure all the user entered values are okay.